rails/react-components/src/library/components/permission-forms-v2/permission-form-row.tsx

```diff
@@ -2,15 +2,28 @@ import React from "react";
 import css from "./style.scss";
 import { IPermissionForm } from "./permission-form-types";
 
-
 interface PermissionFormRowProps {
   permissionForm: IPermissionForm;
 }
 
+function ensureUrlProtocol(url: string): string {
+  if (!url) {
+    return "";
+  }
+  // Regular expression to check if the URL starts with http:// or https://
+  const urlPattern = /^(http:\/\/|https:\/\/)/i;
```

Comment:
Do you think you should maybe trim the url first before checking this pattern, eg `if (!urlPattern.test(url.trim())) {` or make the pattern `/^\s*(http:\/\/|https:\/\/)/i`?  Any leading spaces will cause the test to fail and return an url like `https:// https://example.com`.

Reply:
Yes, it definitely makes sense, thanks! Fixed in https://github.com/concord-consortium/rigse/pull/1319/commits/066171959f6ed0026ac306d68bff0ca9a52e9974